lib/Dist/Zilla/Dist/Builder.pm

```diff
@@ -511,7 +510,7 @@ sub build_archive {
              ? '_build_archive_with_wrapper'
              : '_build_archive';
 
-  my $archive = $self->$method($built_in, $basename, $basedir);
+  my $archive = $self->$method($built_in, $basedir);
```

Comment:
If there is a good reason to pass in the `$basename` I am fine doing that, but as mentioned in the commit I noticed neither Dist::Zilla nor my example plugin (updated here: https://github.com/uperl/Dist-Zilla-Plugin-Libarchive/blob/f5e705a67d765f2299c7b1743de114a22a76b47a/lib/Dist/Zilla/Plugin/Libarchive.pm) were using it, and the `Dist::Zilla` methods are private and it should be fine to change the signature.